### DISCUSSION TITLE
When will PRs be allowed?

### DISCUSSION THREAD

**Discussion by flexagoon** [deckerst/aves#159] [General]
You currently do not accept pull requests, do you plan to allow them any time soon?

**Comment by deckerst** [comment]
Sorry to disappoint, but I don't plan to allow them for the foreseeable future.

**Reply by flexagoon** [reply]
That's fine, I was just asking. But what's the reason behind that? Do you just think that it'll take too much time to review them? Or is it something else?

**Reply by deckerst** [reply]
Various reasons, but to keep it short I'd say that it's a personal project. Developing it is fun and intellectually stimulating. If people like the app, good for them.

The sources are public to be transparent, and to potentially inspire other developers, but not because I need to collaborate. Otherwise it's just work.

**Reply by flexagoon** [reply]
That makes sense, thanks!
